dev-docs/trees/dbl-000C.tree

```diff
@@ -0,0 +1,91 @@
+\title{Modules over models of double theories}
+\import{macros}
+
+\p{A right module over a model of a double theory is a special kind of bimodule out of #{1}. Recall the definition of bimodule from ([Lambert &
+Patterson 2024](cartesian-double-theories-2024), Definition 9.1). In loc. cit. the word module is used where we use bimodule here. The aim here is to define a notion of module that is to arbitrary bimodules as, in the case of categories (models of the terminal double theory), #{C}-sets and presheaves are to arbitrary profunctors. An extra constraint is necessary to achieve this, as we see in the example below:}
+
+\transclude{dbl-000B}
+
+\p{In the case of multicategories, we'd like to recover plain multifunctors into \Set, which
+are known to be a good notion of copresheaf for various purposes. To this end we make the following definition:}
+
+\subtree{
+    \title{Right and left modules over a model}
+    \taxon{definition}
+
+    \p{Let \dbl{D} be a double theory and let \dbl{E} be a double category with a terminal object. A \define{right module} over a model #{X} of \dbl{D} in \dbl{E} is a bimodule #{H:1\proTo X} out of the terminal model #{1} of \dbl{D} in \dbl{E} such that "#{1} acts trivially on the left" in the sense that:
+    \ul{
+    \li{The action cells below are isomorphisms:
+    \quiver{% https://q.uiver.app/#q=WzAsNSxbMCwwLCIxKHgpIl0sWzEsMCwiMSh5KSJdLFsyLDAsIlgoeikiXSxbMCwxLCIxKHgpIl0sWzIsMSwiWCh6KSJdLFswLDEsIjEobSkiLDAseyJzdHlsZSI6eyJib2R5Ijp7Im5hbWUiOiJiYXJyZWQifX19XSxbMSwyLCJYKG4pIiwwLHsic3R5bGUiOnsiYm9keSI6eyJuYW1lIjoiYmFycmVkIn19fV0sWzMsNCwiWChtbikiXSxbMCwzLCI9IiwxXSxbMiw0LCI9IiwxXV0=
+    \begin{tikzcd}
+	{1(x)} & {1(y)} & {X(z)} \\
+	{1(x)} && {X(z)}
+	\arrow["{1(m)}", "\shortmid"{marking}, from=1-1, to=1-2]
+	\arrow["{=}"{description}, from=1-1, to=2-1]
+	\arrow["{X(n)}", "\shortmid"{marking}, from=1-2, to=1-3]
+	\arrow["{=}"{description}, from=1-3, to=2-3]
+	\arrow["{X(mn)}", from=2-1, to=2-3]
+    \end{tikzcd}}
+    }
+    \li{#{X} sends all cells of \dbl{D} to isomorphisms in \dbl{E}.}
```

Comment:
Thanks Kevin, this is a useful writeup!

I'm confusing about this line though. Why should M being a module impose this condition on X? It would be bad if it did because it would seem to break the example of monoidal categories, where a module/instance should recover the notion of monoidal copresheaf, i.e., a lax monoidal functor into (Set, \times).

Reply:
Since F=1 by assumption, I agree that this proves that M(\alpha) is the identity. But in the text you are saying that G=X maps all cells to isomorphisms, which seems like a different statement.